scripts/health_check.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import time
+from enum import Enum
+
+import requests
+
+
+ENV_ENUM = Enum(
+    'ENV',
+    [
+        ('dev', 'https://addons-dev.allizom.org'),
+        ('stage', 'https://addons.allizom.org'),
+        ('prod', 'https://addons.mozilla.org'),
+        # For local environments hit the nginx container as set in docker-compose.yml
+        ('local', 'http://nginx'),
+    ],
+)
+
+
+class Fetcher:
+    def __init__(self, env: ENV_ENUM, verbose: bool = False):
+        self.environment = ENV_ENUM[env]
+        self.verbose = verbose
+
+    def _fetch(self, path: str) -> dict[str, str] | None:
+        url = f'{self.environment.value}/{path}'
+        if self.verbose:
+            print(f'Requesting {url} for {self.environment.name}')
+
+        data = None
+        # We return 500 if any of the monitors are failing.
+        # So instead of raising, we should try to form valid JSON
+        # and determine if we should raise later based on the json values.
+        try:
+            response = requests.get(url, allow_redirects=False)
+            data = response.json()
+        except (requests.exceptions.HTTPError, json.JSONDecodeError) as e:
+            if self.verbose:
+                print(
+                    {
+                        'error': e,
+                        'data': data,
+                        'response': response,
+                    }
+                )
+
+        if self.verbose and data is not None:
+            print(json.dumps(data, indent=2))
+
+        return data
+
+    def version(self):
+        return self._fetch('__version__')
+
+    def heartbeat(self):
+        return self._fetch('__heartbeat__')
+
+    def monitors(self):
+        return self._fetch('services/monitors.json')
```

Comment:
```suggestion
        return self._fetch('services/__heartbeat__')
```

(But first investigate why the 404+error raised in the run I did wasn't causing the health check to fail)